src/openforms/submissions/tests/test_tasks_confirmation_emails.py

```diff
@@ -559,6 +560,71 @@ def test_send_confirmation_email_when_appointment_is_changed(self):
         message = mail.outbox[0]
         self.assertEqual(message.subject, f"Confirmation mail {_('(updated)')}")
 
+    def test_template_is_rendered_in_submission_language(self):
+        language = "en"
+
+        form = make_translated(FormFactory).create(
+            _language=language,
+            name="Translated form name",
+        )
+        FormStepFactory.create(
+            form=form,
+            form_definition__name_en="A Quickstep",
+            form_definition__configuration={
+                "components": [
+                    {
+                        "key": "foo",
+                        "type": "textfield",
+                        "label": "Foo",
+                        "showInEmail": True,
+                    }
+                ],
+            },
+        )
```

Comment:
I don't particularly see the benefit of using `make_translated` here, in tests where we are explicitly testing that the correct languages are used. It's much easier to understand what data is being set up by using the explicit `_en`-suffixed-fields, or maybe even do the test data setup within a `translation.override("en")` context manager.

See also the earlier remarks - we definitely favour using the `FormFactory(generate_minimal_setup=True)` trait to ensure consistent data setup over time - this may now be a correct database state, but it may not be in the future and having the trait + centralized post-generation implementation reduces that risk.